lib/mayaUsd/ufe/MayaUsdContextOps.cpp

```diff
@@ -236,12 +236,89 @@ makeUSDReferenceFilePathRelativeIfRequested(const std::string& filePath, const U
     return relativePathAndSuccess.first;
 }
 
-bool sceneItemSupportsShading(const Ufe::SceneItem::Ptr& sceneItem)
+#ifdef UFE_V4_FEATURES_AVAILABLE
+void addNewMaterialItems(const Ufe::ContextOps::ItemPath& itemPath, Ufe::ContextOps::Items& items)
```

Comment:
New helper function created from code below (just moved here), so I can call it from bulk as well.